src/components/ui/Button/Button.stories.mdx

```diff
@@ -0,0 +1,228 @@
+      }}
+    >
+      <Button inverse {...args}>
+        Button
+      </Button>
+    </div>
+  ) : (
+    <Button {...args}>Button</Button>
+  )
+}
+
+<header>
+
+# Button
+
+A element that allows users to take actions, make choices or trigger an event with a single tap.
+
+</header>
+
+The `Button` component uses [FastStore UI Button](https://www.faststore.dev/reference/ui/atoms/Button) as base.
+
+### Import
+
+`import Button from 'src/components/ui/Button'`
+
+### Usage
+
+<Canvas>
+  <Story
+    name="Button"
+    args={{
+      variant: 'primary',
+      iconPosition: 'left',
+      icon: <Icon name="User" width="18" height="18" />,
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Button" />
+
+| Local token                                  | Default value/Global token linked                                                                 |
+| -------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| <code>--fs-button-padding</code>             | <code>calc(var(--fs-spacing-1) - (var(--fs-button-border-width) \* 2)) var(--fs-spacing-3)</code> |
+| <code>--fs-button-height</code>              | <code>var(--fs-control-tap-size)</code>                                                           |
+| <code>--fs-button-gap</code>                 | <code>var(--fs-spacing-2)</code>                                                                  |
+|                                              |                                                                                                   |
+| <code>--fs-button-shadow</code>              | <code>none</code>                                                                                 |
+| <code>--fs-button-shadow-hover</code>        | <code>none</code>                                                                                 |
+|                                              |                                                                                                   |
+| <code>--fs-button-border-radius</code>       | <code>var(--fs-border-radius-default)</code>                                                      |
+| <code>--fs-button-border-width</code>        | <code>var(--fs-border-width-thick)</code>                                                         |
+| <code>--fs-button-border-color</code>        | <code>transparent</code>                                                                          |
+|                                              |                                                                                                   |
+| <code>--fs-button-text-size</code>           | <code>var(--fs-text-size-base)</code>                                                             |
+| <code>--fs-button-text-weight</code>         | <code>bold</code>                                                                                 |
```

Comment:
```suggestion
| <code>--fs-button-text-weight</code>         | <code>var(--fs-text-weight-bold)</code>                                                                                 |
```

Reply:
Can you also update here pls? Thanks.


https://github.com/vtex-sites/gatsby.store/blob/d2eab5cf901a9969332c97565b091774ae2d1912/src/components/ui/Button/button.scss#L21